app/Http/Controllers/Auth/ForgotPasswordController.php

```diff
@@ -27,6 +27,6 @@ class ForgotPasswordController extends Controller
      */
     public function __construct()
     {
-        $this->middleware('guest');
+        // $this->middleware('guest');
```

Comment:
why is this commented?

Reply:
I use forgot password page for change password function for user (who logs in). If there is this middleware, users who logs in can not access the page.